src/sql/platform/connection/common/connectionStatusManager.ts

```diff
@@ -109,6 +122,15 @@ export class ConnectionStatusManager {
 
 	public onConnectionComplete(summary: azdata.ConnectionInfoSummary): ConnectionManagementInfo {
 		let connection = this._connections[summary.ownerUri];
+		if (!connection) {
+			this._logService.info(`########## OnConnectionComplete but no connection found '${summary.ownerUri}' Connections = [${Object.keys(this._connections)}]`);
```

Comment:
I don't know if we're following the `##########` convention in the our comments anywhere else.  Is there a reason to do so here?